src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable complexity */
+import React from 'react';
+import { withTheme } from 'styled-components/native';
+
+import {
+  Theme,
+} from '../../common/themeSelectors';
+
+import { BadgeContainer, BadgeContent } from './Badge.style';
+
+export type BadgeTypes = 'dot' | 'standard';
+export type BadgeColors = 'alert' | 'primary' | 'secondary' | 'success';
+export type BadgeLimit = 9 | 99 | 9999;
+
+export interface BadgeProps {
+  /**
+   * Badge color: alert, primary, secondary, success
+   * Default: alert
```

Comment:
Se você botar um `@` aqui essa info fica destacada na IDE das usuárias

```suggestion
   * @default: `alert`
```